lib/components/home_page/home_app_bar.dart

```diff
@@ -12,7 +12,7 @@ AppBar homeAppBar(BuildContext context) {
       ),
     ),
     title: const Text(
-      "Chez Soso et Richou",
+      "Clément Bolin",
```

Comment:
don't write Clement Bolin in hardcopy, but in a variable so that it is easier to modify once the api is connected

Reply:
This is a temporary way of doing it since when we will be calling the backend and using our API, this part of the code will change quite a bit to take in account the call to the api but it is still good ot use a variable instead of a hardcopy.